src/features/identity/IdentityAgentManager.ts

```diff
@@ -1,20 +1,21 @@
 import { IdentityAgent } from "@web5/identity-agent";
 import {
   type CreateDidMethodOptions,
-  type ManagedIdentity,
   AppDataVault,
   DwnManager,
+  SyncManagerLevel,
 } from "@web5/agent";
 import {
   MessageStoreLevel,
   DataStoreLevel,
   EventLogLevel,
 } from "@tbd54566975/dwn-sdk-js/stores";
+import { Level } from "level";
```

Comment:
`import type` would make it more clear here that we aren't actually using `Level`